test/testcafe/README.md

```diff
@@ -15,19 +15,19 @@ Run All in headless chrome
 
 Run single Fixture
 
-- `yarn test -t testcafe chrome -f "Unknown user form"`
+- `yarn test -t testcafe chrome --filter "Unknown user form"`
 
 Run all similar Fixtures that matches the pattern
 
-- `yarn test -t testcafe chrome -F "Unknown.*"`
+- `yarn test -t testcafe chrome --filter "Unknown.*"`
 
 Run a single test
 
-- `yarn test -t testcafe chrome -t "should have editable fields"`
+- `yarn test -t testcafe chrome --filter "should have editable fields"`
```

Comment:
how would it identify between -F, -f, -t, -T, --live, --debug-mode ?

Reply:
Right now it's filtering basted on the test name, not the fixture. This should account for most cases and we can address the edge cases when they come up.

In regards to other flags like `--live` and `--debug-mode`, those get captured via the [`additionalArgs` logic](https://github.com/okta/okta-signin-widget/blob/744d2997a0a9a51eeb41f9e140c6001b05ce9844/scripts/buildtools/commands/test.js#L69), and will continue to work.

Reply:
With this change we are loosing flexibility and usefulness of testcafe command line.. We should not enforce use of `testcaferc.json` config on the developer, instead it should be optional..